project.janet

```diff
@@ -0,0 +1,15 @@
+(declare-project
+  :name "janetui"
+  :url "https://github.com/janet-lang/janetui"
+  :repo "https://github.com/janet-lang/janetui.git")
+
+(rule "build/janetui.so" ["CMakeLists.txt"]
+(do
```

Comment:
From this line to the end of the form code should be indented. Btw we have formatting stuff in the https://github.com/janet-lang/spork

Reply:
Got it. I will fmt it in the next commit.